src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -913,6 +939,10 @@ protected DynamoModel(IStartConfiguration config)
                 }
             }
 
+#if DEBUG
+            CurrentWorkspace.NodeAdded += CrashOnDemand.CurrentWorkspace_NodeAdded;
```

Comment:
I think it would be good to unsubscribe this at shudown/dispose.

Reply:
done